stix_shifter_modules/guardium/configuration/config.json

```diff
@@ -23,24 +27,26 @@
             "type": "password",
             "optional": true
         },
-        "client_id": {
-            "type": "text",
-            "previous": "connection.options.client_id"
-        },
-        "client_secret": {
-            "type": "password",
-            "previous": "connection.options.client_secret"
-        }
+            "client_id": {
+                "type": "string",
+                "optional": true,
+                "hidden": true
+            },
+            "client_secret": {
+                "type": "password",
+                "optional": true,
+                "hidden": true
```

Comment:
Not sure why hidden and optional keys are added. Please revert back client_id and client_secret fields to this -
```
       "client_id": {
            "type": "text",
            "previous": "connection.options.client_id"
        },
        "client_secret": {
            "type": "password",
            "previous": "connection.options.client_secret"
        }
```

Reply:
done